README.md

```diff
@@ -453,16 +453,16 @@ This might not be recommended for production, but is good enough for test.
 Container images used in the tests are:
 
 - PostgreSQL:
-  - version 13: `quay.io/bitnami/postgresql:13.4.0`
-  - version 12: `registry.redhat.io/rhscl/postgresql-12-rhel7` (only if `ts.redhat.registry.enabled` is set)
+  - version 13: `postgres:13.6`
+  - version 10: `registry.redhat.io/rhscl/postgresql-10-rhel7` (only if `ts.redhat.registry.enabled` is set)
```

Comment:
why does this postgresql version need to be changed?

Reply:
It is not required, but we usually update to the latest minor

Reply:
And some of our tests used 13.4, others — 13.5 and one still uses 9-something 🙂

Reply:
Regarding 12->10: we do not use version 12 anywhere. We use registry images of versions 13 and 10, but this was not reflected in the docs.